client/www/pages/dash/index.tsx

```diff
@@ -320,19 +320,26 @@ function Dashboard() {
       return;
     }
 
+    const isAppIdValid = Boolean(apps.find((a) => a.id === appId));
+    if (appId && isAppIdValid) return;
+
     const firstApp = apps?.[0];
     if (!firstApp) return;
 
-    const isAppIdValid = Boolean(apps.find((a) => a.id === appId));
-    if (appId && isAppIdValid) return;
+    const lastAppId = getLocal('dash_app_id');
+
+    const defaultAppId = lastAppId ?? firstApp.id;
```

Comment:
Would it be possible here that `lastAppId` is no longer valid? 

i.e someone deletes that app. 

We may want to do a check for that